mcp/communications/tests/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+find_package(cmocka CONFIG)
+
+# EVTM code
+
+add_executable(test_evtm
+    ${CMAKE_CURRENT_LIST_DIR}/test_evtm.c
+)
+
+add_dependencies(test_evtm libcmocka)
+
+target_include_directories(test_evtm PRIVATE
+    ${CMOCKA_INCLUDE_DIR}
+    ${CMAKE_CURRENT_LIST_DIR}/..
+    ${PROJECT_SOURCE_DIR}/testing # get mock mcp defines
+)
+target_link_libraries(test_evtm PRIVATE
+    ${CMOCKA_LIBRARY}
+    communications
+)
+
+# last element SHOULD NOT have a comma, else you will get a linker error
+# "/usr/bin/ld: cannot find : No such file or directory"
+string(CONCAT LINK_FLAGS_WRAP
+    "-Wl,"
+    "--wrap=initBITSender,"
+    "--wrap=setBITSenderSerial,"
+    "--wrap=setBITSenderFramenum,"
+    "--wrap=sendToBITSender,"
+    "--wrap=testing_evtm,"
```

Comment:
I suggested updating this function name, so it would have to be updated here as well